services/opencollective/opencollective-base.spec.js

```diff
@@ -0,0 +1,37 @@
+import { expect } from 'chai'
+import nock from 'nock'
+import { cleanUpNockAfterEach, defaultContext } from '../test-helpers.js'
+import OpencollectiveBase from './opencollective-base.js'
+
+class DummyOpencollectiveService extends OpencollectiveBase {
+  static route = this.buildRoute('dummy')
+
+  async handle({ collective }) {
+    const data = await this.fetchCollectiveInfo({
+      collective,
+      accountType: [],
+    })
+    return this.constructor.render(this.getCount(data))
+  }
+}
```

Comment:
No concerns with using a dummy child class, though it does remind of a scenario we ran into with some of the GitLab classes a ways back where we forgot to wire up the auth for one or two of them. As a thought/potential future exercise, I wonder if there's a clean way we could consistently test "does every class that's supposed to use the auth" without having to write all the boilerplate for every single class :thinking:

Reply:
Yeah it is a fair point, but the pattern here is not not unique to this service or PR. We use the same approach in other places e.g: https://github.com/badges/shields/blob/master/services/stackexchange/stackexchange-base.spec.js

I've raised https://github.com/badges/shields/issues/9493 as I agree it is worth improving